packages/cloud-cognitive/src/components/InlineEdit/InlineEdit.test.js

```diff
@@ -75,4 +76,197 @@ describe(componentName, () => {
       componentName
     );
   });
+
+  it('can start and stop edit with focus', async () => {
```

Comment:
You don't need the async as you're not doing any awaits.

```suggestion
  it('can start and stop edit with focus', () => {
```